packages/design-system-scripts/package.json

```diff
@@ -15,6 +15,7 @@
     "@babel/core": "^7.8.4",
     "@babel/plugin-proposal-class-properties": "^7.12.1",
     "@babel/plugin-transform-object-assign": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.13.6",
```

Comment:
Why is this dependency added or needed?

Reply:
The use of `async` for `export async function sendTealiumEvent` is giving referenceError of `regeneratorRuntime is not defined`, and this plugin fixes that. Is there another package that can address this issue? Or alternative is that we don't use async, just regular send and re-try like hrtools.

Reply:
You shouldn't need an additional plugin for this. If you search for `async` in our codebase you can see it's already being used in a bunch of different places like in e2e tests. If you want to use `async` you just need to include `import "regenerator-runtime/runtime";` (i.e.  in `e2e.global-setup.js` which is run before each e2e test). You can see [this stackoverflow](https://stackoverflow.com/a/61517521) for more info

Reply:
That being said, I dont know if we want to be introducing `regenerator-runtime/runtime` as a dependency to our library. Since we already use `core-js`, it might be better to use promises instead of `async` and just include a `core-js` polyfill for promise support. You can see Tooltips as an example of this